internal/jobs/stats/stats.go

```diff
@@ -88,6 +100,81 @@ func StartCollector(bufSize uint, jc types.JobConfiguration) *StatsCollector {
 		logrus.Infof("Capabilities: %v", s.ReportedCapabilities)
 	}
 
+	// --- Twitter Key Type Capabilities ---
+	if tamIface, ok := jc["twitter_account_manager"]; ok {
+		if tam, ok := tamIface.(*twitter.TwitterAccountManager); ok {
+			var hasBase, hasElevated, hasCredential bool
+			for _, key := range tam.GetApiKeys() {
+				switch key.Type {
+				case "base":
+					hasBase = true
+				case "elevated":
+					hasElevated = true
+				case "credential":
+					hasCredential = true
+				}
+			}
+			if hasBase {
+				s.ReportedCapabilities = append(s.ReportedCapabilities, "twitter_base")
+			}
+			if hasElevated {
+				s.ReportedCapabilities = append(s.ReportedCapabilities, "twitter_elevated")
+			}
+			if hasCredential {
+				s.ReportedCapabilities = append(s.ReportedCapabilities, "twitter_credential")
+			}
+		}
+	}
+
+	// --- Twitter Key Type Capabilities ---
+	// Try to detect Twitter key types if TwitterAccountManager is available
+	if tamIface, ok := jc["twitter_account_manager"]; ok {
+		if tam, ok := tamIface.(interface {
+			GetApiKeys() []*twitter.TwitterApiKey
+		}); ok {
+			var hasBase, hasElevated, hasCredential bool
+			for _, key := range tam.GetApiKeys() {
+				switch key.Type {
+				case "base":
+					hasBase = true
+				case "elevated":
+					hasElevated = true
+				case "credential":
+					hasCredential = true
+				}
+			}
+			if hasBase {
+				s.ReportedCapabilities = append(s.ReportedCapabilities, "twitter_base")
+			}
+			if hasElevated {
+				s.ReportedCapabilities = append(s.ReportedCapabilities, "twitter_elevated")
+			}
+			if hasCredential {
+				s.ReportedCapabilities = append(s.ReportedCapabilities, "twitter_credential")
+			}
+		}
+	}
+
+	logrus.Info("Starting stats collector")
+
+	s = stats{
+		BootTimeUnix: time.Now().Unix(),
+		Stats:        make(map[statType]uint),
+	}
+	for _, t := range allStats {
+		s.Stats[t] = 0
+	}
+
+	capabilities, isString = jc["capabilities"].(string)
+	if isString {
+		if strings.Contains(capabilities, ",") {
+			s.ReportedCapabilities = strings.Split(capabilities, ",")
+		} else {
+			s.ReportedCapabilities = []string{capabilities}
+		}
+		logrus.Infof("Capabilities: %v", s.ReportedCapabilities)
+	}
+
```

Comment:
```suggestion
	if capabilities, isString = jc["capabilities"].(string); isString {
		s.ReportedCapabilities = strings.Split(capabilities, ",")
		logrus.Infof("Capabilities: %v", s.ReportedCapabilities)
	}

```

`strings.Split` will return a single-element slice if the separator is not found. It might be a good thing to also apply this in https://github.com/masa-finance/tee-worker/pull/62/files#diff-b7e738eff2627c4e6e40af4b35a6f10a7efd6586b1259d999a22754f7e07f18bR93-R102